libs/content/workspace-ffi/Cargo.toml

```diff
@@ -13,6 +13,7 @@ egui = "0.26.2"
 egui_wgpu_backend = "0.28"
 libc = "0.2"
 egui_editor = { path = "../editor/egui_editor" }
+egui_extras = { version = "0.26.2", features = ["image"] }
```

Comment:
installing image loaders is now an integration concern instead of a widget concern